### PR TITLE
Luxury Bluespace Shelter Capsules, Take 2.

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
add: Luxury versions of the bluespace shelter capsule are now available! Purchase them at the mining equipment vendor.
add: Cardboard cutouts have a new option: Xenomorph Maid
add: Black Carpet can now be crafted using a stack of carpet and a black crayon.
add: Black fancy tables can now be crafted using Black Carpet.
add: Shower curtains can now be recoloured with crayons, unscrewed from the floor, disassembled with wire cutters, and reassembled using cloth, plastic, and a metal rod.
/:cl:

take7

So my first PR is completely fucked because I'm an idiot and can't into git. I closed that one, and now I've made this one.  It includes everything you see in this image, and a bar of deluxe soap in the shower, and a survival knife hidden in the toilet.  The meal is random for each survival pod, and so is the potted plant. The display case has a random fake item that you can take out and use to prank the crew/get arrested by security for.

### CODE REVIEW

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -54,6 +54,35 @@
...
+			/obj/item/weapon/reagent_containers/food/snacks/bearsteak,
+			/obj/item/weapon/reagent_containers/food/snacks/enchiladas,
+			/obj/item/weapon/reagent_containers/food/snacks/stewedsoymeat,
+			/obj/item/weapon/reagent_containers/food/snacks/burger/bigbite,
+			/obj/item/weapon/reagent_containers/food/snacks/burger/superbite,
+			/obj/item/weapon/reagent_containers/food/snacks/burger/fivealarm)
+
+/obj/effect/spawner/lootdrop/three_course_meal/Initialize(mapload)
+	loot = list(pick(soups) = 1,pick(salads) = 1,pick(mains) = 1)
+	..()
```
Comment (ChangelingRain) [inline code/game/objects/effects/spawners/lootdrop.dm]: . = ..()

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 2]
```diff
@@ -106,6 +106,11 @@
 	turf_type = /turf/open/floor/carpet
 	resistance_flags = FLAMMABLE
 
+/obj/item/stack/tile/carpet/black
+	name = "black carpet"
+	icon_state = "tile-carpet-black"
```
Comment (ChangelingRain) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: I think you missed adding this icon state too

Comment (Expletive) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: Oh woops. yeah. I got it now though. Next commit.

File: code/modules/mining/shelters.dm [hunk 3]
```diff
@@ -40,3 +40,18 @@
 /datum/map_template/shelter/alpha/New()
 	. = ..()
 	whitelisted_turfs = typecacheof(/turf/closed/mineral)
+
+
+/datum/map_template/shelter/beta
+	name = "Shelter Beta"
+	shelter_id = "shelter_beta"
+	description = "An extremly luxurious shelter, containing all \
```
Comment (SamHPurp) [inline code/modules/mining/shelters.dm]: "extremely" </pedant>

**Pull request merged**